Sources/RoutingKit/AnyRouter.swift

```diff
@@ -0,0 +1,55 @@
+/// A router that performs type erasure by wrapping another router.
+public struct AnyRouter<Output>: Router {
```

Comment:
Not in love with these docs, but mostly stole them from [`AnyPublisher`](https://developer.apple.com/documentation/combine/anypublisher) in Combine. Would welcome any better suggestions.